subgraphs/friend-tech/src/mappings/handlers.ts

```diff
@@ -41,7 +40,7 @@ export function handleTrade(event: Trade): void {
     event
   );
 
-  const tradeID = createTrade(
+  createTrade(
```

Comment:
nit: we can remove the return from `createTrade()` as this is the only function using it